website/src/content/concepts/integrate-with-your-own-api.mdx

```diff
@@ -31,6 +31,7 @@ Requests to that endpoint should additionally pass the following HTTP headers:
 - `Accept-version` (**required**): See [Versioning](#versioning).
 - `X-Forward-To` (**required**): Set it to the external API URL.
 - `X-Forward-To-Audience-Policy` (**required**): See [Configuring the audience policy](#configuring-the-audience-policy).
+- `X-Forward-To-Claims` (optional): See [Configuring custom claims](#configuring-custom-claims)
```

Comment:
```suggestion
- `X-Forward-To-Claims` (optional): See [Configuring custom claims](#configuring-custom-claims).
```

Reply:
Updated in this commit: 088fb79b95c5c8d8d309617db4f8f9f1a47da13b